templater/main.go

```diff
@@ -0,0 +1,177 @@
+package main
```

Comment:
Can we avoid copying this from distro and instead share it somehow?

Reply:
@jaxesn this file is not same as distro file. I made some changes to build EKS-A specific jobs.

Reply:
maybe we can start like this, since i think it would be better to get this merged sooner rather than waiting, but we could look at how we could better share this or turn it into a library that can be reused, even if there are specific eks-a vs eks-d capabilities?